packages/mds-compliance-service/@types/index.ts

```diff
@@ -46,6 +46,14 @@ export interface ComplianceSnapshotDomainModel {
   total_violations: number
 }
 
+/**
+ * If a generated snapshot can't be written to Kafka, make a note for later cleanup
+ */
+export interface ComplianceSnapshotKafkaFailureDomainModel {
```

Comment:
Can we make this less Kafka specific and have it just be `ComplianceSnapshotFailureDomainModel`? I can imagine that one day there may be other types of snapshot failures that could use this same mechanism. Also, we may not always use Kafka.